examples/dynamic/views/Content.js

```diff
@@ -1,29 +1,35 @@
 var React = require('react');
 
-var TodoList = React.createClass({
-  render: function() {
+class TodoList extends React.Component {
+  constructor(props) {
```

Comment:
Let's leave the constructor out entirely. It's not needed if all you call is `super()`.